client/landing/jetpack-cloud/components/daily-backup-status/index.jsx

```diff
@@ -178,13 +187,11 @@ class DailyBackupStatus extends Component {
 
 		return (
 			<>
-				<Gridicon icon="cloud-upload" className="daily-backup-status__gridicon-no-backup" />
-
-				<div className="daily-backup-status__date">
-					{ translate( 'No backups are available yet.' ) }
+				<div className="daily-backup-status__message-head">
+					<img src={ cloudWarningIcon } alt="" role="presentation" />
+					<div>{ translate( 'No backups are available yet' ) }</div>
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.